Elastic.Console/Elastic.Console.psm1

```diff
@@ -824,12 +830,28 @@ function Invoke-Elasticsearch {
                 Set-Variable -Name $ResponseVariable -Value $response -Scope Global
             }
 
-            if ($response.Content -is [string]) {
-                #PowerShell
-                return $response.Content
-            } else {
+            if ($response.Content.GetType().FullName -eq "System.Net.Http.HttpContent") {
                 #PowerShell Core
-                return $response.Content.ReadAsStringAsync().Result;
+                if ($Bytes) {
+                    return $response.Content.ReadAsByteArrayAsync().Result;
+                } else {
+                    return $response.Content.ReadAsStringAsync().Result;
```

Comment:
Not sure how PS reacts to .Result being in a console app, is it worth waiting on AsyncWaitHandle (with a non-zero timeout to avoid pegging a core) ?

Reply:
I believe PowerShell's Synchronization context is safe to perform this operation